Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -43,16 +44,39 @@ protected theorem _root_.Ideal.smul_eq_mul (I J : Ideal R) : I • J = I * J :=
   rfl
 #align ideal.smul_eq_mul Ideal.smul_eq_mul
 
+variable (R M) in
```

Comment:
The files Ideal/Operations and Ideal/QuotientOperations are long, maybe some split will have to be considered…

Reply:
or golfs ...